ansible/configs/migrating-to-ocpvirt/pre_software.yml

```diff
@@ -77,34 +77,40 @@
       include_role:
         name: bastion-student-user
 
-    - name: Copy credentials to host temporarily
-      template:
-        src: ./files/rfc2136.ini.j2
-        dest: /home/lab-user/.rfc2136.ini
+
+    - name: Write Route53 credentials into /root/.aws/credentials
+      blockinfile:
+        dest: "/root/.aws/credentials"
+        create: true
+        owner: "root"
+        mode: 0600
```

Comment:
I think we prefer symbolic mode as it is less error prone and easier to read? `u=rw,g=,o=`